scripts/src/main/resources/scripts/command/aws

```diff
@@ -71,45 +67,26 @@ function doPackageInstall()
 }
 
 function doSetup() {
-  local ERR=""
-  findAws
-  if [[ ${AWS_INSTALLED} != "yes" ]] || [[ "${1}" != "silent" ]]
+  local silent="${1}"
+  doInstall "aws" "${AWS_VERSION}" "${silent}" "" "${DEVON_IDE_HOME}/updates/install/aws"
+  if [ "${?}" = 0 ]
   then
-    # Get leatest release
-    if [ -z "${AWS_VERSION}" ]
-    then
-      doEcho "Getting latest release..."
-      AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
-    fi
-
-    doInstall "-" "${DEVON_IDE_HOME}/updates/install/aws" "aws" "${AWS_VERSION}"
-    ERR="${?}"
-    if [ "${ERR}" = "0" ]
-    then
-      doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws" "${AWS_VERSION}"
-    fi
-    doRunCommand "rm -rf ${DEVON_IDE_HOME}/updates/install/aws"
```

Comment:
Should we keep this after `doPackageInstall`?
The problem is that we then do not have `.devon.software.version` file if we remove it.
When we keep it, `doInstall` will return `1` if the same version was already present and installed.
Otherwise on every setup `aws` will be reinstalled or we need to re-add more custom logic for AWS.

Reply:
I resolved this and kept the line to remove it.
@CREITZ25 has found an excellent solution to unpack MSI files. We will create a separate story to extend `doExtract` so that it can also unpack MSI files just like tar, zip, oder dmg files. Maybe we can also find a solution for extracting PKG files on Mac.
Here I simply try to keep compatibility with previous behavior.